go/vt/vtctl/vtctl.go

```diff
@@ -229,6 +229,10 @@ var commands = []commandGroup{
 				"<tablet alias>",
 				"Changes metadata in the topology server to acknowledge a shard master change performed by an external tool. See the Reparenting guide for more information:" +
 					"https://github.com/vitessio/vitess/blob/master/doc/Reparenting.md#external-reparents."},
+			{"PreExternallyReparentShard", commandPreExternallyReparentShard,
```

Comment:
I think it will be simpler to have repairReplication check with Orchestrator if it's in the middle of a maintenance instead. There are a few advantages:
* Orchestrator has one less dependency.
* You don't have to worry about the lifecycle of this state. What if Orc is not able to update this after it sets the value?